extensions/spring-data-jpa/deployment/src/main/java/io/quarkus/spring/data/deployment/SpringDataJPAProcessor.java

```diff
@@ -1,18 +1,18 @@
 package io.quarkus.spring.data.deployment;
 
+import static java.util.stream.Collectors.*;
```

Comment:
We don't do start imports.

Reply:
You can configure your IDE to not automatically replace multiple imports with star imports

Reply:
Ah, yes!! I knew it :woman_facepalming: I was a bad configuration in my IDE. Now it's ok.